### PR TITLE
Restructure proposal 003-IoT-Adoption

### PR DESCRIPTION
**Pull request by choeppler**
Restructure the IoT-Adoption proposal according to the template provided
and comment out sections that have not been covered by the proposal.

### CODE REVIEW

File: design/003-IoT-Adoption.md [hunk 1]
```diff
@@ -10,29 +10,31 @@
...
 
 ## Details
 
-Separating the functionality to reduce the needed available implementation and communication on a device. In general, the idea of a watchtower [3] is to react if the corresponding entity is offline or loses connectivity. Targeting IoT devices, the general assumption is that these devices have limited capabilities in regard to connectivity. Therefore, the watchtower or a watching function will run separately.
+Separating the functionality to reduce the needed available implementation and communication on a device. As shown in Figure 2, the following main components should be available as independent components:
```
Comment (manoranjith) [inline design/003-IoT-Adoption.md:23]:
```suggestion
Separating the functionality to reduce the needed implementation and communication on a device. As shown in Figure 2, the following main components should be available as independent components:
```

**Review by manoranjith — CHANGES_REQUESTED** [review]
@choeppler  I have just one suggestion. Also, I think we can also wrap the text at 80 chars, so that it is easy to read when opening in a text editor as well.

File: design/003-IoT-Adoption.md (hunk 1 above)
Comment (choeppler) [reply design/003-IoT-Adoption.md]: Rephrased in 049c6332

**Comment by choeppler** [comment]
@manoranjith, I didn't introduce linebreaks so actual changes can be seen. Can do so in a separate commit before merging after the review is done.

**Comment by manoranjith** [comment]
> 
> 
> @manoranjith, I didn't introduce linebreaks so actual changes can be seen. Can do so in a separate commit before merging after the review is done.

Okay. LGTM.

**Review by manoranjith — APPROVED** [review]

**Comment by choeppler** [comment]
@manoranjith , I didn't see your comment before merging #13. So I opened another PR #15 with the format change.

**Pull request merged**